pkg/bot-code/client.go

```diff
@@ -0,0 +1,151 @@
+I'll refactor the function signatures in pkg/bot-github/client.go to use unique structs for arguments where there are more than two parameters.
```

Comment:
apply these change requests to pkg/bot-github/client.go NOT a new file at pkg/bot-code/client.go